tools/quake3/q3map2/game_qfusion.h

```diff
@@ -110,7 +110,7 @@
 	"scripts",          /* shader directory */
 	65535,              /* max lightmapped surface verts */
 	65535,              /* max surface verts */
-	393210,             /* max surface indexes */
```

Comment:
What is the nature of overflow here? q3map2 code or engine? I guess this was value copypasted from engine code.

Reply:
I don't know where it came from. I think it was updated when the sRGB support was added, but I wasn't around. Indexes are usually verts * 6 and I asume someone did that without putting much thinking into it